packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx

```diff
@@ -46,12 +55,25 @@ export const BarChartStyling = () => {
             <Config>
                 <Config.Section>
                     <Config.Heading>{`Y-axis label`}</Config.Heading>
-
                     <TextInput
                         defaultValue={yAxisLabel}
                         radius="md"
                         onChange={(e) => onYAxisLabelChange(e.target.value)}
                     />
+                    <Select
+                        data={['left', 'right']}
+                        defaultValue={yAxisPosition}
+                        onChange={(value) =>
+                            onYAxisPositionChange(value || undefined)
+                        }
+                        placeholder={'Change the y-axis position'}
+                    />
+                </Config.Section>
+            </Config>
+            <Config>
+                <Config.Section>
+                    <Config.Heading>{`Series`}</Config.Heading>
+                    <TextInput radius="md" />
```

Comment:
do you want to keep this here? The `series` bit

Reply:
I'll remove this!